mobile/src/components/TeamFooter/index.js

```diff
@@ -1,8 +1,10 @@
-import { Image, StyleSheet, Text, View } from 'react-native'
+import { Animated,  Image, StyleSheet, Text, View } from 'react-native'
+import {useRef, useEffect} from 'react'
 import { scale, verticalScale } from 'react-native-size-matters'
 import { colors, fontFamilies, fonts, fontWeights } from '../../utils/theme'
 
-const TeamFooter = ({ icon, name, totalScore }) => {
+const TeamFooter = ({ icon, name, totalScore, isAnswerCorrect }) => {
```

Comment:
Is `isAnswerCorrect` used anywhere in the file? Looks like it's added but not used

Reply:
Not currently, but I think we'll need it when we add in the animation for a correct answer (which is on the list). I'm removing it for now for clarity and will add it back when it's actually required.